scipy/special/add_newdocs.py

```diff
@@ -3073,25 +3073,64 @@ def add_newdoc(name, doc):
     """)
 
 add_newdoc("fresnel",
-    """
-    fresnel(z)
+    r"""
+    fresnel(z, out=None)
 
-    Fresnel sin and cos integrals
+    Fresnel integrals.
 
-    Defined as::
+    The Fresnel integrals are defined as
 
-        ssa = integral(sin(pi/2 * t**2), t=0..z)
-        csa = integral(cos(pi/2 * t**2), t=0..z)
+    .. math::
+
+       S(z) &= \int_0^z \cos(\pi t^2 /2) dt \\
+       C(z) &= \int_0^z \sin(\pi t^2 /2) dt.
+
+    See [dlmf]_ for details.
 
     Parameters
     ----------
-    z : float or complex array_like
-        Argument
+    z : array_like
+        Real or complex valued argument
+    out : 2-tuple of ndarrays, optional
+        Optional output arrays for the function results
 
     Returns
     -------
-    ssa, csa
-        Fresnel sin and cos integral values
+    S, C : 2-tuple of scalar or ndarray
+        Values of the Fresnel integrals
+
+    See Also
+    --------
+    fresnel_zeros : zeros of the Fresnel integrals
+
+    References
+    ----------
+    .. [dlmf] NIST Digital Library of Mathematical Functions
+              https://dlmf.nist.gov/7.2#iii
+
+    Examples
+    --------
+    >>> import scipy.special as sc
+
+    As x goes to infinity, S and C converge to 0.5.
```

Comment:
Is the `x` here the `z` in the integrals & parameters above? That seems to be the case from the formula you've linked to from DLMF. Maybe that's obvious enough, but it had me thinking for a bit.

Reply:
Oops, changed everything to use `z` and specify that the limit is taken along the real axis.